packages/shared-component--membershipHero/contentful/.eslintrc.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  rules: {
+    'func-names': [
```

Comment:
Could these rules be moved into a shareable dependency and used with extends, that way if we were to have to make an update to rules it would be a minor version update?

It would keep consistency across the components with regards to code style, but might be overkill, what do you think?

Reply:
@SampsonM Good thinking, something like this?

```
.eslintrc.js
.eslintrc.contentful.js
```

Then all those **./contentful** dir configs can extend the root `.eslintrc.contentful.js`?